BrainPortal/app/models/demand.rb

```diff
@@ -60,8 +33,10 @@ class Demand < ActiveRecord::Base
                         :institution, :department, :position, :email,
                         :city, :province, :country, :confirm_token
 
-  validates             :login, :length => { :minimum => 3, :maximum => 20 },     :allow_blank => true
-  validates             :login, :format => { :with => /^[a-zA-Z][a-zA-Z0-9]+$/ }, :allow_blank => true
+  validates             :login,
```

Comment:
This is just duplicating the rules in the User model, please implement a validator method here that delegates to the ones in the User model...